webapi/src/main/scala/org/knora/webapi/messages/util/PermissionUtilADM.scala

```diff
@@ -955,5 +902,65 @@ object PermissionUtilADM extends LazyLogging {
 
     maybePermissionLevel.map(_.toInt)
   }
+}
+
+trait PermissionUtilADM {
+
+  /**
+   * Given a permission literal, checks that it refers to valid permissions and groups.
+   *
+   * @param permissionLiteral the permission literal.
+   * @return the validated permission literal, normalised and reformatted.
+   */
+  def validatePermissions(permissionLiteral: _root_.java.lang.String): zio.Task[_root_.java.lang.String]
```

Comment:
`_root_.java.lang` types are by intention right?

Reply:
oh, no. it is my IDE refactoring being too explicit. 

Will fix it in a followup PR.

Reply:
If this should be just  Scala `String` i can also change it in mine PR, since it responder I do now is also dependent on `ResourceUtilV2`.